ramp/test/test_run.py

```diff
@@ -23,8 +23,8 @@ def series_to_average(profile_series, num_days):
         
         average_series = profile_series[0:1440]
         for d in range(1,num_days):
-            average_series += profile_series[0+1440*d:1440+1440*d].set_index(average_series.index)
-            average_series = average_series/ (d+1)
+            average_series = average_series + profile_series[0+1440*d:1440+1440*d].set_index(average_series.index)
```

Comment:
We should replace this to use numpy broadcasting rather than looping over the days, but later